lib/Command/Configure.php

```diff
@@ -210,6 +210,17 @@ private function configureSms(InputInterface $input, OutputInterface $output) {
 				$providerConfig->setDid($did);
 				break;
 
+			case 'huawei_e3531':
+				$config->setProvider($provider);
+				/** @var HuaweiE3531Config $providerConfig */
```

Comment:
You have to add a `use` stmt for this class to import it

Reply:
done